transport/src/main/java/io/netty/channel/nio/AbstractNioChannel.java

```diff
@@ -125,12 +131,51 @@ protected SelectionKey selectionKey() {
         return selectionKey;
     }
 
+    /**
+     * No longer supported.
+     */
+    @Deprecated
     protected boolean isReadPending() {
         return readPending;
     }
 
-    protected void setReadPending(boolean readPending) {
-        this.readPending = readPending;
+    /**
+     * No longer supported.
+     */
+    @Deprecated
+    protected void setReadPending(final boolean readPending) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)